lib/aws-record/record/item_operations.rb

```diff
@@ -81,6 +81,24 @@ def save(opts = {})
         end
       end
 
+
+      # Performs mass assignment of model attributes
+      def assign_attributes(opts)
+        opts.each do |field, new_value|
+          if !respond_to?("#{field}=")
+            raise ArgumentError.new "Invalid field: #{field} for model"
+          end
+          @data.set_attribute field, new_value
+        end
+      end
+
+      # Updates model attributes and then performs a conditional save
+      def update(opts)
```

Comment:
We should also add an `#update!` method, which essentially does the same thing except it raises an exception when the validation fails. Check the implementation of `#save!`